### PR TITLE
Overview connector fixes

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

File: pages/OverviewPage.qml [hunk 1]
```diff
@@ -146,11 +146,6 @@ SwipeViewPage {
 		for (let i = 0; i < widgets.length; ++i) {
 			const connectors = widgets[i].connectors
 			for (let j = 0; j < connectors.length; ++j) {
-				// For all left widgets except for solar (which has two connectors), straighten the
-				// connector if there is only one widget (as it will expand to full size).
-				if (widgets === _leftWidgets) {
-					connectors[j].straight = connectors.length === 1 && _leftWidgets.length === 1
```
Comment (DanielMcInnes) [inline pages/OverviewPage.qml:152]: `WidgetConnector.straight` appears to be unused now

File: data/AcInputs.qml [hunk 2]
```diff
@@ -44,13 +44,27 @@ QtObject {
...
 		}
 	}
 
+	function sourceValid(source) {
+		return source !== VenusOS.AcInputs_InputSource_NotAvailable && source !== VenusOS.AcInputs_InputSource_Inverting
+	}
+
+	function findValidSource() {
+		if (activeInput) {
+			return activeInput.source
```
Comment (DanielMcInnes) [inline data/AcInputs.qml:59]: the return value of this function is only used when it is AcInputs_InputSource_NotAvailable. Might be clearer if the function was `bool hasValidSource()`?

**Review by DanielMcInnes — APPROVED** [review]
lgtm, nits

File: pages/OverviewPage.qml (hunk 1 above)
Comment (blammit) [reply pages/OverviewPage.qml:152]: Thanks, removed

File: data/AcInputs.qml (hunk 2 above)
Comment (blammit) [reply data/AcInputs.qml]: It was to accomodate changes in https://github.com/victronenergy/gui-v2/pull/948 that need `hasValidSource()`, but yeah probably best to keep this change separate. Have updated this to be `hasValidSource()`.

**Pull request merged**